Makefile

```diff
@@ -699,6 +699,17 @@ SPRACINGF3MINI_SRC	 = \
 		   $(HIGHEND_SRC) \
 		   $(COMMON_SRC) \
 		   $(VCP_SRC)
+		   
+IRCFUSIONF3_SRC = \
+		   $(STM32F30x_COMMON_SRC) \
+		   drivers/accgyro_mpu.c \
+		   drivers/accgyro_mpu6050.c \
+		   drivers/barometer_bmp085.c \
+		   drivers/flash_m25p16.c \
+		   drivers/serial_softserial.c \
```

Comment:
softserial needed? really? I'm kinda suprised because there's no `user` IO on the Vortex Fusion PCB, right?  Or am I missing something?